recipes-connectivity/networkmanager/networkmanager-1.22.10/always-assume-uid-for-external-processes.patch

```diff
@@ -3,14 +3,17 @@
 #       Hence assume uid to be 0(root).
```

Comment:
If that is pending work, it should be isolated in its own branch/PR.
IIUC, you re-introduced the recipes used in `zeus` in https://github.com/OpenXT/xenclient-oe/pull/1355/commits/4776be455030d6b9c72826132b77bb5f495c1eb0 and https://github.com/OpenXT/xenclient-oe/pull/1355/commits/879e60d009e2d488620fdecc039e7e6aa0ea7937.